code/game/gamemodes/wizard/artefact.dm

```diff
@@ -199,5 +211,5 @@
 
 /obj/item/weapon/scrying/attack_self(mob/user as mob)
 	user << "\blue You can see...everything!"
-	visible_message("\red <B>[usr] stares into [src], their eyes glazing over.</B>")
+	visible_message("\red <B>[user] stares into [src], their eyes glazing over.</B>")
```

Comment:
Text macro and <B>? Would be best to replace it with `<span class='danger'>message</span>`.